File: python_ToSXs/python/02-caudieukien/79.py
year = 2000
month = 1
day = 0

match month:
    case 1 | 3 | 5 | 7 | 8 | 10 | 12:
        day = 31
    case 4 | 6 | 9 | 11:
        day = 30
    case 2:
        if year % 400 == 0 or (year % 4 == 0 and year % 100 != 0):
            day = 29
        else:
            day = 28
print(f"Tháng {month} năm {year} có {day} ngày.")

File: cpp_e9zMZ/cpp/02-caudieukien/79.cpp
#include <iostream>
using namespace std;

int main() {
    int month = 2;
    int year = 2022;
    int result = 0;

    switch (month) {
        case 1:
        case 3:
        case 5:
        case 7:
        case 8:
        case 10:
        case 12:
            result = 31;
            break;
        case 4:
        case 6:
        case 9:
        case 11:
            result = 30;
            break;
        case 2:
            if (year % 400 == 0 || (year % 4 == 0 && year % 100 != 0)) {
                result = 29;
            } else {
                result = 28;
            }
            break;
    }

    printf("Tháng %d năm %d có %d ngày.", month, year, result);
    return 0;
}